src/client/components/pages/parts/pager.js

```diff
@@ -46,6 +46,10 @@ class PagerElement extends React.Component {
 			// eslint-disable-next-line react/no-did-update-set-state
 			this.setState({from: 0, query: this.props.query}, this.triggerSearch);
 		}
+		if (this.props.refreshTable) {
```

Comment:
While it looks to me like this pattern would work, I don't think it's the most straightforward solution, and might generate extra rendering in the process.

To be honest there's no ideal way to communicate punctual events from parent to child, but what I usually end up doing is creating a reference to the child component and calling `myChildRef.doSomething()` from the parent component.
That way, the logic of deciding what should happen and when stays with the parent where it belongs, while the details of what's supposed to happen stay with the child.

Here's an example where we use that pattern:
https://github.com/bookbrainz/bookbrainz-site/blob/7ef08d9d5ca84f7e8befa6230117c5232d606c70/src/client/components/input/entity-search.js#L80
https://github.com/bookbrainz/bookbrainz-site/blob/7ef08d9d5ca84f7e8befa6230117c5232d606c70/src/client/components/input/entity-search.js#L38

Reply:
In this case, in the parent (CollectionPage) it would look something like this:
```
closeAddEntityModalShowMessageAndRefreshTable(message) {
		this.setState({
			message,
			showAddEntityModal: false
		}, this.pagerElementRef.triggerSearch();
	}
```